lib/split/dashboard/public/style.css

```diff
@@ -312,10 +312,13 @@ a.button.green:focus, button.green:focus, input[type="submit"].green:focus {
   background:#768E7A;
 }
 
-#filter, #clear-filter {
+.dashboard-controls input, .dashboard-controls select {
   padding: 10px;
 }
 
+.dashboard-controls-bottom {
+  margin-top: 10px;
+}
```

Comment:
Already in our code: https://github.com/clio/split/blob/master/lib/split/dashboard/public/style.css#L315-L321